oemof_flexmex/plotting/prepare.py

```diff
@@ -0,0 +1,180 @@
+    r"""
+    Sums imports and exports of electricity by copying all relevant rows from the Dataframe,
+    adding the values together and then replacing the rows by their sum in the original Dataframe.
+
+    Parameters
+    -------------
+    plot_data: pandas.Dataframe
+    scenario: str
+    one single scenario, e.g. "FlexMex2_2b"
+    region: str
+    one single region, e.g. "DE"
+
+    Returns
+    ------------
+    plot_data: pandas.Dataframe
+    the input dataframe with summed transmission
+    """
+    df_total_outgoing = plot_data[
+        (plot_data.loc[:, "Parameter"] == "Transmission_Flows_Electricity_Grid")
+        & (plot_data.loc[:, "Scenario"] == scenario)
+        & (plot_data.loc[:, "Region"].str.contains(region + "_"))
+    ]
+
+    total_outgoing = -df_total_outgoing["Value"].sum()
+    row_total_outgoing = {
+        "Scenario": scenario,
+        "Region": region,
+        "Parameter": "Transmission_Outgoing",
+        "Unit": "GWh",
+        "Value": total_outgoing,
+    }
+    df_total_incoming = plot_data[
+        (plot_data.loc[:, "Parameter"] == "Transmission_Flows_Electricity_Grid")
+        & (plot_data.loc[:, "Scenario"] == scenario)
+        & (plot_data.loc[:, "Region"].str.contains("_" + region))
+    ]
+    total_incoming = df_total_incoming["Value"].sum()
+    row_total_ingoing = {
+        "Scenario": scenario,
+        "Region": region,
+        "Parameter": "Transmission_Incoming",
+        "Unit": "GWh",
+        "Value": total_incoming,
+    }
+
+    plot_data.drop(df_total_outgoing.index.to_list(), inplace=True)
+    plot_data.drop(df_total_incoming.index.to_list(), inplace=True)
+    plot_data = plot_data.append(
+        row_total_outgoing, ignore_index=True
+    )  # is represented by shifted bottom line
+    plot_data = plot_data.append(row_total_ingoing, ignore_index=True)
+
+    return plot_data
+
+
+def make_losses_negative(plot_data):
+    r"""
+    Searches for rows that report losses in the Dataframe and makes their values negative.
+
+    Parameters
+    -------------
+    plot_data: pandas.Dataframe
+
+    Returns
+    -------------
+    plot_data: pandas.Dataframe
+    """
+    for i in range(
+        len(plot_data)
+    ):  # TODO: This is very bad style. Generally, one shouldn't iterate over rows. -> improve
+        if "Losses" in plot_data.loc[:, "Parameter"].iloc[i]:
+            plot_data.iloc[i, plot_data.columns.get_loc("Value")] = (
+                plot_data.iloc[i, plot_data.columns.get_loc("Value")] * -1
+            )
+    return plot_data
+
+
+def prepare(plot_data, scenario, region, object, df_demand=False):
+    r"""
+    Transforms the input dataframe into a dataframe that has the expansion steps as rows and the parameters as
+    columns. Furthermore, it extracts the relevant demands from the demand file, depending on the scenario.
+
+    Parameters
+    -------------------
+    plot_data: pandas.Dataframe
+    scenario: str
```

Comment:
Please revisit the numpy docstring format.

```suggestion
    Parameters
    -------------------
    plot_data : pandas.Dataframe
        Description
    scenario : str
         Description
```